main.py

```diff
@@ -0,0 +1,21 @@
+from flask import Flask, render_template
+import json
+
+# initializes the flask app
+app = Flask(__name__)
+
+@app.route("/")
+def homepage():
+    """"Renders the index page of the app."""
+    return render_template("index.html")
+
+@app.route("/data/<file_name>")
+def send_data(file_name):
+    """Sends the requested file."""
+    with open('./'+str(file_name),"r") as json_file:
```

Comment:
This certainly works for local development, but using relative pathing with an un-sanitized file name would be a security concern in the wild. Your frontend is only requesting legitimate files, but what if the file_name were `../../../documents/something_you_dont_want_to_share`? Rather than a file_name, can the frontend request a dataset ID which you could use to open `./data/charts-<id>.json`?